packages/edit-site/src/components/sidebar-navigation-screen/index.js

```diff
@@ -40,10 +40,15 @@ export default function SidebarNavigationScreen( {
 					) : (
 						<div className="edit-site-sidebar-navigation-screen__icon-placeholder" />
 					) }
-
-					<div className="edit-site-sidebar-navigation-screen__title">
-						{ title }
-					</div>
+					{ ! parentTitle ? (
```

Comment:
Is there any reason to limit the H2 to only the top level?

Reply:
It is to avoid using the h2 as a wrapper for both the text and the add new button on the inner screen.
It is valid HTML, but when using Voiceover the heading text and button is announced twice.

Reply:
Should we introduce a dedicated prop for the "new button", like an "actions" prop for instance?

Reply:
To place it outside of edit-site-sidebar-navigation-screen__title?

Reply:
yes that's the idea